pkg/api/getVariationName.go

```diff
@@ -78,7 +78,7 @@ func (vwo *VWOInstance) GetVariationName(campaignKey, userID string, option inte
 		utils.LogMessage(vwo.Logger, constants.Error, getVariationName, message)
 		return ""
 	}
-	if !utils.CheckCampaignType(campaign, constants.CampaignTypeVisualAB) {
+	if utils.CheckCampaignType(campaign, constants.CampaignTypeFeatureRollout) {
```

Comment:
are you sure this new condition is right?

Reply:
https://github.com/wingify/vwo-python-sdk/blob/master/vwo/api/get_variation_name.py
Line 78: 
If Campaign.Type is FEATURE_ROLLOUT then it should return else continue